.github/workflows/cd.yml

```diff
@@ -0,0 +1,18 @@
+name: cd
+on:
+  workflow_run:
+    workflows: ["ci"]
+    branches-ignore: ["*"]
+    types:
+      - completed
+  push:
+    tags:
+      - "^v.*"
+
+permissions:
+  contents: read
+
+jobs:
+  plugin-cd:
+    uses: mattermost/actions-workflows/.github/workflows/plugin-cd.yml@main
+    secrets: inherit
```

Comment:
The cd workflow should not be in the starter template. This repo is community focused and the CD pipeline is Mattermost specific.

Reply:
good let me remove this.